index.d.ts

```diff
@@ -187,6 +195,8 @@ declare class Client {
     indices<TResponse = Record<string, any>, TContext = Context>(callback: callbackFn<TResponse, TContext>): TransportRequestCallback
     indices<TResponse = Record<string, any>, TContext = Context>(params: RequestParams.CatIndices, callback: callbackFn<TResponse, TContext>): TransportRequestCallback
     indices<TResponse = Record<string, any>, TContext = Context>(params: RequestParams.CatIndices, options: TransportRequestOptions, callback: callbackFn<TResponse, TContext>): TransportRequestCallback
+    // cat.master api is deprecated.
+    // toDo: delete cat.master api when it is removed from OpenSearch
     master<TResponse = Record<string, any>, TContext = Context>(params?: RequestParams.CatMaster, options?: TransportRequestOptions): TransportRequestPromise<ApiResponse<TResponse, TContext>>
```

Comment:
same here I believe you can mark these [deprecated](https://jsdoc.app/tags-deprecated.html)

Reply:
sure. thanks for the link